docker/prepare-environment.sh

```diff
@@ -15,3 +19,5 @@ fi
 
 # Enable ding PHP configuration.
 phpenmod ding
+
+mkdir -p /root/.ssh && touch /root/.ssh/known_hosts && ssh-keyscan -H github.com >> /root/.ssh/known_hosts && chmod 600 /root/.ssh/known_hosts
```

Comment:
@arne I don't quite grok your response here. Isn't `ssh-keyscan` a secure way of fetching the current SSH key signature? Sure, it may be even safer to hardcode them, but then what if they change?

We could also strip them from the DOM on that page, but that is probably considered a hack .. :)